components/navigation.js

```diff
@@ -681,62 +681,13 @@ function NavigationExpanded({ navigationClick, toggleNavCollapsed }) {
           >
             Audit Documentation
           </NavigationHrefItem>
-        </NavigationGroupItem>
-
-        <NavigationGroupItem
-          id="resources"
-          title="Resources"
-          icon={<Icon.Resources />}
-          isGroupOpened={openedParents.includes("resources")}
-          handleGroupClick={handleParentClick}
-        >
           <NavigationHrefItem
             id="overview"
```

Comment:
Change the `id` to `resources`. It doesn’t make a technical difference, but might as well keep things consistent.